src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages = Math.ceil(total / perPage);
+  const pages = [];
+
+  for (let page = 1; page <= numberOfPages; page += 1) {
+    pages.push(
+      <li className={classNames('page-item', { active: page === currentPage })}>
+        <a
+          data-cy="pageLink"
+          className="page-link"
+          href={`#${page}`}
+          onClick={() => onPageChange(page)}
+        >
+          {page}
+        </a>
+      </li>,
+    );
+  }
```

Comment:
```suggestion
  const numberOfPages = Math.ceil(total / perPage);

  const pages = getNumbers(1, numberOfPages);
```

And than you can map throught pages and create `li`